docs/topics/voice.md

```diff
@@ -45,8 +45,7 @@ client.on('message', async message => {
 
 ## Streaming to a Voice Channel
 
-In the previous example, we looked at how to join a voice channel in order to obtain a `VoiceConnection`. Now that we
-have obtained a voice connection, we can start streaming audio to it.
+In the previous example, we looked at how to join a voice channel to obtain a VoiceConnection. Now that we have obtained a voice connection, we can start streaming audio to it.
```

Comment:
Why have you removed the code markdown on `VoiceConnection`? Otherwise the only change I can see here is that you changed the line length.